docs/src/main/tut/verbs.md

```diff
@@ -0,0 +1,47 @@
+---
+menu: verbs
+weight: 115
+title: REST Verbs
+---
+
+For a REST API, your service will want to support different verbs. Http4s has a
+list of all the [verbs] you're familiar with, and a few more.
+
+```tut:book
+import fs2.Task
+import io.circe.generic._
+import io.circe.syntax._
+import org.http4s._, org.http4s.dsl._
```

Comment:
Stylistic question: why these two imports together, and the third one not?  Not a major deal, and certainly legal.  Just wondering if there's a practical advantage.